src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -93,11 +106,48 @@ public void removePerson(Person key) {
         persons.remove(key);
     }
 
+    //// tag-level methods
+    /**
+     * Returns true if a tag with the same identity as {@code tag} exists in the address book.
```

Comment:
May I know what is "identity" in this case? It is possible to change address book to FoodRem :D

Reply:
Will update the javadocs, it's the same name in this case for tags.